src/main/user-impl/java/com/mysql/cj/jdbc/ha/plugins/failover/FailoverConnectionPlugin.java

```diff
@@ -918,35 +898,16 @@ private ConnectionUrlParser.Pair<String, Integer> getHostPortPairFromHostPattern
     return pair;
   }
 
-  private String getRdsClusterHostUrl(String host) {
-    Matcher matcher = auroraDnsPattern.matcher(host);
-    String clusterKeyword = getClusterKeyword(matcher);
-    if ("cluster-".equalsIgnoreCase(clusterKeyword)
-        || "cluster-ro-".equalsIgnoreCase(clusterKeyword)) {
-      return matcher.group(1) + ".cluster-"
-          + matcher.group(3); // always RDS cluster endpoint
-    }
-    return null;
-  }
-
-  private String getRdsInstanceHostPattern(String host) {
-    Matcher matcher = auroraDnsPattern.matcher(host);
-    if (matcher.find()) {
-      return "?." + matcher.group(3);
-    }
-    return null;
-  }
-
   private void identifyRdsType(String host) {
-    this.isRds = isRdsDns(host);
+    this.isRds = rdsHelper.isRdsDns(host);
```

Comment:
which syntax would you prefer for that call?

`rdsHelper.isRdsDns()` or `RdsUtils.isRdsDns()`